devutils/update_submodule.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+UNGOOGLED_REPO=$(dirname $(dirname $(readlink -f ${BASH_SOURCE[0]})))/ungoogled-chromium
+
+# Ensure the submodule is initialized.
+git submodule init
+git submodule update
+
+# Pull latest commits and get the tag submodule is on.
+cd $UNGOOGLED_REPO
+CURRENT_TAG=$(git describe)
+git pull -q
+cd ..
+
+CHROMIUM_VERSION=$(git --git-dir=ungoogled-chromium/.git show origin/master:chromium_version.txt)
+UNGOOGLED_REVISION=$(git --git-dir=ungoogled-chromium/.git show origin/master:revision.txt)
+
+if [ UNGOOGLED_REVISION != "0" ]
+then
+    UPDATED_TAG="${CHROMIUM_VERSION}-${UNGOOGLED_REVISION}"
+else
+    UPDATED_TAG="${CHROMIUM_VERSION}"
+fi
+
+# Update the submodule.
+cd $UNGOOGLED_REPO
+if [ $CURRENT_TAG == $UPDATED_TAG ]
+then
+    echo "Submodule already on latest version."
+else
+    git checkout $UPDATED_TAG
+    cd ..
+    git add ungoogled-chromium
+    echo "Submodule updated. Commit the change when you can."
```

Comment:
This message will erroneously show in the following situation:

1. Submodule's `master` and latest tag point to the same commit
2. ungoogled-chromium repo updates `master` with new commits, but no new tag is created.
3. Someone tries to update this repo using this script.

But this probably won't be a probem in most cases, so it's up to you if you want to fix it.